server/controllers/trustController.js

```diff
@@ -0,0 +1,43 @@
+const trustController = {};
+const pool = require('../database/database.js');
+const { check, validationResult } = require('express-validator');
+const config = require('../../config/config.js');
+const log = require("loglevel");
+log.setLevel("debug");
+const TrustModel = require("../models/TrustModel");
+const asyncHandler = require('express-async-handler');
+const expect = require("expect-runtime");
+
+const asyncUtil = fn =>
+function asyncUtilWrap(...args) {
+  const fnReturn = fn(...args)
+  const next = args[args.length-1]
+  return Promise.resolve(fnReturn).catch(e => {
+    console.error("get error:", e);
+    next(e);
+  })
+}
+
+trustController.get = async (req, res, next) => {
+  const trustModel = new TrustModel();
+  res.locals.response = {
+    trust_relationships: trustModel.get(),
+  }
+  next();
+};
+
+trustController.request = asyncUtil(async (req, res, next) => {
```

Comment:
use asyncUtil to wrap the express handler, so no need to use structure: try..catch to handle error, including the error throwing from internal layer (model)